aws-android-sdk-iot/src/main/java/com/amazonaws/mobileconnectors/iot/AWSIotMqttManager.java

```diff
@@ -172,6 +172,8 @@
      */
     private static final String SDK_VERSION = VersionInfoUtils.getVersion();
 
+    protected static int portNumber = 8883;
```

Comment:
Does this need to be an instance variable? Better to create an internal method that accepts a port number as a parameter.

Reply:
Removed this variable